src/plugins/wizard/public/application/components/top_nav.tsx

```diff
@@ -58,10 +58,10 @@ export const TopNav = () => {
         appName={PLUGIN_ID}
         config={config}
         setMenuMountPoint={setHeaderActionMenu}
-        showSearchBar={true}
-        useDefaultBehaviors={true}
-        screenTitle="Test"
```

Comment:
Do we still need a defined screen title or does it default to the app name or something?

Reply:
it's an optional parameter that can be the title of the visualization but it can be ignored. When ignored the title is OpenSearch Dashboards. We can add it in future when we have the saved object stuff all polished up.